index.html

```diff
@@ -30,7 +27,9 @@
   <section class="search-section">
     <input type="text" placeholder="Search software jobs" />
     <button class="search-button">
-      <span class="fa fa-search"></span>
+      <svg class="icon">
+        <use xlink:href="images/icons.svg#icon-search"></use>
```

Comment:
Students won't know that. Mentioned in the other PR, and you can add it there, but you need to explain specifically when there should be and shouldn't be a title.